Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/DeletionProcessGracePeriod/HandlerTests.cs

```diff
@@ -0,0 +1,151 @@
+﻿using Backbone.BuildingBlocks.Application.PushNotifications;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Application.Identities.Commands.DeletionProcessGracePeriod;
+using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Application.Infrastructure.PushNotifications.DeletionProcess;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using FakeItEasy;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.Identities.Commands.DeletionProcessGracePeriod;
+public class HandlerTests
+{
+    [Fact]
+    public async Task No_Active_Deletion_Processes()
+    {
+        // Arrange
+        var mockIdentitiesRepository = A.Fake<IIdentitiesRepository>();
+        var mockPushNotificationSender = A.Fake<IPushNotificationSender>();
+
+        A.CallTo(() => mockIdentitiesRepository.FindAllWithApprovedDeletionProcess(A<CancellationToken>._, A<bool>._))
+            .Returns(new List<Identity>());
+
+        var handler = CreateHandler(mockIdentitiesRepository, mockPushNotificationSender);
+
+        // Act
+        await handler.Handle(new DeletionProcessGracePeriodCommand(), CancellationToken.None);
+
+        // Assert
+        A.CallTo(() => mockIdentitiesRepository.FindAllWithApprovedDeletionProcess(A<CancellationToken>._, A<bool>._))
+            .MustHaveHappenedOnceExactly();
```

Comment:
This does not have to have happened. It doesn't matter how the Handler gets the data it needs.